nix/process-compose/cli.nix

```
@@ -70,6 +57,22 @@ in
         default = true;
         description = "Enable or disable the TUI for the application.";
       };
+      arguments = process-compose-flake-lib.mkProcessComposeArgumentsOption { };
```

`cli` is simpler I think.

```suggestion
      cli = process-compose-flake-lib.mkProcessComposeArgumentsOption { };
```